joplin/urls.py

```diff
@@ -23,6 +23,7 @@ def home(request):
 
     path('admin/', home),
     url(r'admin/pages/(\d+)/publish/$', joplin_views.publish, name='publish'),
+    url(r'admin/pages/new_from_modal/$', joplin_views.new_page_from_modal, name='new_page_from_modal'),
```

Comment:
nitpick, maybe we could create a path for admin api routes. so like admin/joplinapi/pages/create or admin/ajax/pages/create -- or something similar?